framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2056,6 +2056,25 @@ void VulkanReplayConsumerBase::InitializeScreenshotHandler()
 
     if (!options_.screenshot_dir.empty())
     {
+        if (util::filepath::Exists(options_.screenshot_dir))
```

Comment:
You've already committed this change to `dev`.  I think it's harmless (squash merge will probably just omit it) but it's confusing that it's listed as part of this PR. Is it possible you had not updated `dev` on your system before you made this branch?

Reply:
Please rebase this branch on current `dev` so this change isn't listed as part of this PR.

Reply:
Quite possible.  Just clicked update-branch below.